cubi_tk/irods/check.py

```diff
@@ -103,16 +106,32 @@ def get_irods_error(cls, e: Exception):
         es = str(e)
         return es if es != "None" else e.__class__.__name__
 
-    def get_data_objs(self, root_coll: iRODSCollection):
+    def get_data_objs(
+        self, root_coll: iRODSCollection
+    ) -> typing.Dict[
+        str, typing.Union[typing.Dict[str, iRODSDataObject], typing.List[iRODSDataObject]]
+    ]:
         """Get data objects recursively under the given iRODS path."""
-        data_objs = dict(files=[], checksums={})
+        data_objs = dict(files=[], checksums=[])
```

Comment:
this should be `data_objs = dict(files=[], checksums={})` (dict for checksums), no?